From 7e8ac587a47204645dab185c071d6921a0cd2d60 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 29 Oct 2026 09:06:59 +0100
Subject: [PATCH 1/2] Update the upstream signing key

Refresh gpgkey-5AE420CC0209989E.asc from keyservers/WKD: same
certificate(s), with self-signatures our copy was missing.
The commands are in the comment above its Source line.

The current key is needed to verify the release signature with
openpgpverify (sqv), which checks that the key was valid when the
release was signed.

Assisted-by: Claude Opus 5.5
---
 gpgkey-5AE420CC0209989E.asc | 556 +++++++++++++++++++++++++++++++-----
 znc.spec                    |   4 +
 2 files changed, 484 insertions(+), 76 deletions(-)

diff --git a/gpgkey-5AE420CC0209989E.asc b/gpgkey-5AE420CC0209989E.asc
index bc8a0f5..bc1b4a4 100644
--- a/gpgkey-5AE420CC0209989E.asc
+++ b/gpgkey-5AE420CC0209989E.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQINBE66oWABEADT/ix/xviuWpwAeySocHnYozbjO4L1lKzaj4OMfAZfP+KvwX8H
+xsFNBE66oWABEADT/ix/xviuWpwAeySocHnYozbjO4L1lKzaj4OMfAZfP+KvwX8H
 jy0Ymf4KwxVkub0R2znLlmYiSjHaelMNwK9EDaY48uodV7EMHBostP6u+tjUJQsZ
 tQI3ErUYOv0Qlx3xfZ8UvlCT6AWdvlZQha8oAs3EzaUPrBD5dsnUFE6wq0UsBBoA
 5xCh/2eGEqKjtSNTf1ZtSCF1StfRPPX8bqkEMpBsTTak3G6+3yt9GYRAqDwSTkB2
@@ -11,79 +11,483 @@ nns4oyQqfjRGyCC9kw7hmUDQ0LBMPaoxAf5IOuXQlLc7RHmdcwvML6nBzIV2ueJG
 AMWnNh4G8Ms4N6buZHHAipZ8EzL9itbNVjvKdh2K02wq6M8/QvRKlp1kcmpJsDAC
 501tUXMyPA1K9FppWRrXjmAzYvSSXXRcXT8U6RXxVA48JI6UaO4AL2iV1MjmsXo6
 SdaPCFg4oiziTGgCdNcdKqSJU7N8oYDwHRZ7wSOeVI1b7ZB6cXbrQAHBUQARAQAB
-tCNBbGV4ZXkgU29rb2xvdiA8a3RvbmlidWRAZ21haWwuY29tPokCPgQTAQIAKAIb
-AwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AFAlhytLQFCQ9brdAACgkQWuQgzAIJ
-mJ4kYA/7Bmak/BOdHNoSQCGzAj+rTiWNuTVmaMPU9cIrDvh2VvpLMHZfVktQ7pEv
-lKZh4/93SXKY9MrFg2U5ThFO+O5wd62S0dTmUZuzh54qvxTQgCA1ShgCvDj+6YGn
-IyrbtkLOCEiFT1+GR2pBdaZfsiO1uGZ177KOR7TSoh95ZA7/euU0nWX+75DTtaLK
-vbwG0/H9vF1/ANAVsGMQEw2a/L5JDc0UxUYDwydkLoow+JVzSwYmhS5uLr5+bSv8
-i2uvcFOiLktBosu/Mrt4vOvzMdUlA1Tto8E945CZdRFahU7LHHnMRZseqy/928S8
-L7MLuCnyCIF1B6tJecoXgjaAtyRFfwhnpZbx5r9ebsT6nHLXSWmGoQCT8AbaP+pu
-tQTeT0exzf0JY5kLosKT0XdklGHyuOpOlid+kyEJXTgLZBkpBW9Ee4Qu9NzTvaBM
-Et8bM7d9pj56e6i+ybMYAOm9eM8wP6aUGrSmMlm1d1k2X4PiwyrLui7pJ/2oRgqX
-DGwGXEi4dEjyrXMPAjqFTKJUzYG5+DkmUohD4WBD+SYyZnxeLhu5AJDsQlBA+cRn
-yluy6ohCr2Pp+32bIVzYZzl6uTT+WcIuY7d8eF8c/9lvG63rCHn10xOC5/En+CCX
-bjCP2l0XBMaoC1EXeakoeoIsp0uyOzKVz2eciL+wVSZ2+Iq3US20JEFsZXhleSBT
-b2tvbG92IDxhbGV4ZXlAYXNva29sb3Yub3JnPokCQQQTAQIAKwIbAwYLCQgHAwIG
-FQgCCQoLBBYCAwECHgECF4ACGQEFAlhytLAFCQ9brdAACgkQWuQgzAIJmJ7Xmg/9
-EZexxioWUbpNMBUbXMLjPWQXxyV+/53nDGnl8WJWvPStIbubulratXRJtL8J8C1t
-uTMgSP85bRsnUxv1ETOwg7esET13LR3sqDDWIzvqFiiupNBh3CI54Ub761JsZBCk
-rtiKjg6EDwOK3hCCaQt3RdATeLAF/Qxok8WAHEgWhCd0sjSlhMyqTiMvFpod678A
-WYoOjPbb3Vc+pdl7qC0hBaIwKgjnW3eowjoSWDg8VyUSb1vtwXyknKHZ1ZmzOjjx
-TRE1Rw69NQMzJCEMZaF1L5JdolQbS4dm27NRN1Fd5hgXhfe7doq/FLd/f8sgNvIL
-UctdzpI3HNmN5QXH2xaDkY3fFc2e+8qHK6wPOX2IKMBTBXwnXFxJIQcDXEImVo8U
-1sZtiABan2Yfec0mnVO/ofiYrR7z5hrFnUfuTUs3XUWUmJSqzPmf43zbRbTndhlL
-nLdHiRz35+cFhleh1mDgFbFRQCtA7a7NF0CIcAWZxFe6WtcTb83J4Dq+6tsO01hS
-9b3npYAjV6pB2nQJAlDH4a9YOZB/fY//Lc00kt8UnTV1NFw3usUBaSq5wIgB3IvS
-RA38FHd+yH05R79nvZUkjtyHiE9znUbehBIJjtB//A3e6Bga9i91ebQBaN/MLa8c
-zbOjP641cLe41/WEDsHe+qP4oNunjJ3bHBvw5F+0RYW0K0FsZXhleSBTb2tvbG92
-IDxhbGV4ZXlAYWxleGV5c29rb2xvdi5jby5jYz6JAh8EMAEIAAkFAlZoQfoCHSAA
-CgkQWuQgzAIJmJ5F0A//VLE/Eg2RzIA1u6bj2F/+q3vVxZGh27druc70/lyERHvF
-zuEpwedWJY9hn8QW8hfoSEf+KF5FR7DQ31l5072en6G4k8uNrVhwWXdraNMV+FoF
-jiFbUPhWu09tHiyqtuPrxVnibPXWqZ66/gta1gbc+5jxm0MA+13czka3DOm26/Vg
-2IIt6GCGM4nUaBSC2iifbVH/FbxHc7KlzK+itdqigp5Ij2qg1tRY/W6t5zFuiiax
-YE6ni9sUljRz/nO+rk5I/ZcTy69rtVmy0z3we58jsxkzDs1SSJ6Qw3gjcMdsmllu
-FFGMm3X6P1X23pjJ7aPzVMkmD5BQEKKGm/Od184i/VGm6a2tai4eHvIn8/VBSNu2
-PqNpCXmL0GnO2d869hXqtVd5EqegaHVEaSLnyqKnxU6Fgnf33epvhkfJ6mdFmnHF
-7RFr/VuGnWRHnkT9ZtGUazBAywv041tIqUMcw22ER2exRei5n/cS9FOfqM7EqfO4
-BKm8RAjtJUzHB7ha/MRj6hQLsBf8mpNZJWKLhOzuvqveTljuGaD6xfTvxZ8zxCRO
-dZrer5VvQkkV35V+e4nJGRAy4Mn0g38C8VF+BbvzIa0Noes3ZJ36Nt/Xuvy+wL4E
-fmBvxLtEs/DFAiTCwX9NAujjDVXlTA81A/Ig7/vpD+g+F2EN9MjtUBTdPod6Gyq0
-LkFsZXhleSBTb2tvbG92IChaTkMpIDxhbGV4ZXkrem5jQGFzb2tvbG92Lm9yZz6J
-Aj8EEwECACkCGwMHCwkIBwMCAQYVCAIJCgsEFgIDAQIeAQIXgAUCWHK0tAUJD1ut
-0AAKCRBa5CDMAgmYniG7D/9C3eM//ZX7O4e4tPdgFQqA8CX286dISnN74Lq0451P
-xw5MpbJvC8sEDcBg3EdGngRF6W8GRQnMHZKhqrVZn/LbzcE1b+CxyjmCT4xXWOJ1
-0vUgaJkJNQGg3VLl4ImhWgRX16jX+poKa85NsbVWm0zl2U9ZxOfQIh+TCQizju7L
-6fK6knYG3rw51YWLWJ268KgxsGgovMVF4e/g6vaFVwZCmhREL9nq8/tDcMJnnwtt
-sAdWHT5axRJOHru3oroatyjnQygsoboMe9ScKC+VZSt27l6fsF6bfct6sqIahW1U
-hV/MajWF/PjdcQBg+BUE/jMzmiB2modl1savr0/O4wjvnRHQjmh5U2VVwJ4YMGVo
-y6TTurEL0POialzIMqLhrGtqv1bIefk7d4FAqGg97gX1HsBOKKWhmXzdBMgR0ueK
-lwjlbYEETya9XvQ2gDQS8Fa0G+RbbXesJLChl07O32ujfzkUlzJnaGSS8l3gV328
-g7+PT5iu9ZBQlLl+X+EGhDhhb/qRKBf0eUHIFmDvVBQ/FTW3SkeKuR4wgQ6z8Sfr
-dyS/Dc9yQKNz7wXsUEzYhLV5eNdBrbFcUOHqXfc/LYzMXGKL78fNUFuZLxs7lw8p
-fb6dNBZ8DqsgbMmCXiboNI+J1IUSLhaXYMlW7mLpq17+U8BpfY/QUVzNgN3lrnJH
-yLkCDQROuqFgARAAprAiO4ddX7PCYekdLhCmpTl6FQRsM9pfObBOjeenMeZ9R0jA
-IiaaoNkAIHQjZFlcUDezzjzoegGqhy/Hf8Zk4rwJ9ypfg4tjKdWtnWPG4NjXMjXV
-vyiaGFl9JwOLV3tXhcLRHRkl8RKE1Sl5ez3SJ7cw3NbEgeJ2esRHjcFULQbQpg4I
-ND6ati/vUOKmP9v/ocpwuUL/2yBZzwPETc6/jeK5MZPtxWBD6zoyoCFmo/0SEORF
-IyUoNQsev5+AtNowQ4ERXhYPBbJkkbrk2usQZDlSaqAcDg3xu4Ln5ft26CT8BJhr
-R6l4WiqPrR9Q92e+Mo/mC+UKL4cmUZeOGX1f9z1niNNjKAhwgjPFAwiahLf5WVmR
-V0mbTeD2mdnJXBam9UYo8WaagmJZYQ4Gk+Vk8akWnO2EUP3baUFsN80wAUlsKIWX
-A7By19A80CiSMrhCkIlgCJY1U9+WWJShGybKv4YuYuizNi8zHOiWIKzulXRtkpHl
-XcKaoYdQz3hv5iPOr+SNSDo3IKLuuNhEAtOreTkuvjASz3X0etGjAqbv8jOCBSH3
-PlNB4BEwdGY5KAEZdARXPm2cv50PW1oamO7B1+ZXaPGM6iH7bZqFaGyBIrBSetnz
-QyaEuTx4ES858atajsEe+74h1Mkhy3zjqdDFkznKNmiMtK3ea2sZCcnqM6EAEQEA
-AYkCJQQYAQIADwIbDAUCWHK01wUJD1ut9wAKCRBa5CDMAgmYnsJaD/0dOb3gYBHX
-P+Sdh/QeH3UhaIQgKvBqFkQs5xXV11aQ3GYlqe2FSMXfqSYx2LLUmonxsLSw8bQb
-HbaQgs2/innLpghTea9Pu1aJYmDa85RFi4vruhht8k7k9fdJLY+L3r6+uL0o463I
-MZZhBFitBpvCabJUw0DUv0ZUp6FfuRA99VwYXy+MbtY6+EqdcF3t/pmiYOcXWPEt
-hJAeJ2zwxF+8dA/uvXjfVyb6NAbMuYyHYzaZmCYW+Bz3+LS8ct3QDOqi2L0cj3vb
-pLcOQD2WzS1tMqx4o8GzVCn0W0246y3rMURYBrd/4wGpSLogbyPk/HbUH+UqVmrO
-HeVXUp88oP91o/nU+8zP5y02Yc3JqllqT8275m1w3NuSwaTa6PiHCmT542OfpBNp
-PuhQ7IPPmwK5unOgFtQH8EydnMopuQjQeNHgrFKmfifn2++aTYnUdOV16qVy4oEv
-aBjF+RqFDkTtSbB4ji72JCpEAtiXStouSWU9K8eIi+Su5J31/jBJV8xyf+MZ3aAE
-By3K3LSty7l4Ie7XqsMD8HMxXUQUj7rUEoysw1YZALeEIR7mzEW5+go+Lq/F71RP
-GcrDrdtKA5xNJZpoOB2IgS2ZBTlrjOdu9QGX8sZdr9I5SNsATtmOvlYHCsKiasQj
-yyIpbKrNT4ugmrTrkOXU85APrub821Rp+g==
-=HB3n
+zS5BbGV4ZXkgU29rb2xvdiAoWk5DKSA8YWxleGV5K3puY0Bhc29rb2xvdi5vcmc+
+wsGWBBMBCABAAhsDBwsJCAcDAgEGFQgCCQoLBBYCAwECHgECF4AWIQTVgjystHcZ
+HKwAdVVa5CDMAgmYngUCZ7UeCgUJImB+KgAKCRBa5CDMAgmYnsYQD/9qrz1WLi/A
+V7gvTuf9CTPG2msThVeNRYXHSXvRGLVc2tDOSkOno4tT2ckB1XZxJINRHy5s8Ycn
+sUBj/nr2Yp7Mxnbo0at58h7stcD+j1x3nJJY+puBoK8I2qbsL6ZFXsEr2WAcomBN
+S0DN0q+knW7IrTUe1Ubip2jutlcHH4+Iji8MMvlZS/DYjZHMwfy3oMLlAqWHlEZa
+oRI8/HmpuwCr+y8tPY6cJg5WwwFspPv+R16EdD+knYl0SY+6ibNUWEjoC6ZomKXd
+qG6mHJ3Szy1UrvRhh4/HYvlmKrOxrg9K4jtDNPwKbyLZ9Guc5Pu5wPQWYJ8ldxGS
+NEar9Xql3RfFj411Z9tz/M6Nrp8Fkbm4b6ZOuAvG6ggFUPyj4HIRMZdNgaZL0T/2
+y0G08DyWVpthnUMMledOMfw/NnLrrE11XjXB9cnfMA2uNHG+V0DjSHPYUPyyeMQA
+SaN1/qIXlVLcklM3jpUDRmD7fugmtObN8axhhdKhYe6YEetw/KTP4t2zTZI/hcMI
+CQ+lvRPpnp29zwjcgHPSiY1+RR193Uw2nxrHmUcfFuynOeMGIv9eYnX7O+w+85Dt
+IoIbEfiWMBRZmnkWMphxJJ54o4JDrOJ5pp89qikMb8lKBiIWHUdz4bGBimAup+St
+ugJm6dA5kg18gUBTr0py6Cd8WkIo+fiFvsLBlgQTAQgAQAIbAwcLCQgHAwIBBhUI
+AgkKCwQWAgMBAh4BAheAFiEE1YI8rLR3GRysAHVVWuQgzAIJmJ4FAmOnlGQFCRqQ
+jYAACgkQWuQgzAIJmJ7msw/+LCBhRnMvGFPHApKZP6Q2tupjoSFPFzWZcdIz1lGi
+8j/WwsqZTdHzh0/0eM3uAbC+fAU0r9e1eCtqVhUHYrS8dqj19+5T6DqCprn8A54s
+RaBQICfHNwSMGGICh/QkwCGgU+G+pSCTRI//8xcazUrWq2us7tDhHhozS/6d/lwl
+bGSrhg90K3mtoT88F9CqfFT8BmX0r5MiLoWiL9zEmd0ejdywP9IAKQHg1PUYC/ZD
+1rNvYOBR+Tupv80/EOI7DQrf7+ttSqe4/7TTkjEy+tlwuMq27xRRXxFProKWJSjv
+H6fTPch/xLwzDhB8ffcbiDkYujuUMskpHkt6uunHvyQs+MakQRr3Wnec21NKK0RY
+lbvwX4eecYHtWerIzVhLEGqj4dP1TjpqBZatqcsK7Mdfd8HSYdR+yUEjk9ozvhM0
+gfSFgJvMET8zTk3xTX2E2X0y/mHL47YCApYPmnp/Mrpi5tIblND9fkpzE3BwbIYB
+GGToT9eOfr8dMQpSKnEo6bgAZLgPe0st8uVGThyy2vVdSOw8MUZP2RCg1GcgOzAw
+Zr2Fo/Zhvyvb4KtflnfZQV95pPaum0hBnvJDGrm56LGX9eiv+rm7bt1YA9LFCzlZ
+MhOqNxQAFl7Rfc5AMlgbiX4Flpgf4Z2s/RJnc1ZntTL3xtLut5Mmu57/Gw9erUOL
+3NPCwZYEEwEIAEACGwMHCwkIBwMCAQYVCAIJCgsEFgIDAQIeAQIXgBYhBNWCPKy0
+dxkcrAB1VVrkIMwCCZieBQJeAAkOBQkU6QIoAAoJEFrkIMwCCZieze0P/3I2OiaK
+apBvpWeDWMtSwB8UnBHtMRNc4KQXkoEkqeWEfyOIvWymS4aWz3xgx0S7EuVZMsaw
+QEIjAFg2xhHmWRoY4Ig+xPGc8fgFY4z4AiKyfeZuzd5Lw6/US0uQUTWkm/ENqHqb
+l0SmRYXBzh/T/Pkh5B8HHH9fkDSj2OfXU+PFt4zALx6u7WMYu9a3WXLY8SucgswH
+xtRKyrviiDib5aEtCp3xHg6Xbep9ySuPup0ie/wNonBZYfGHhyyycFDWvJ8w7vtF
+6+J2DC5tc1AOafR6WbqmdWuaUDSOJui0nUFyoypEGbwWsX4C8yNOv6Ohg7n3unyd
+0DgACAhsBRhI51/oYJrXoSBoUreA3yVPMq2fIJegfrf882y9BkeWTMhrGEdy53AC
+uPDAJSa86lwgmnvmEad2pMhNraAG9rUxJrKEHkhCJRB2Slyi2KNkZImwybwrfK+v
+4xPj5VLNuESlhQWUuJM2LNlcdgg9lS8b4CkiigIGiMUTkRthg8iMqCpDP/N+1YiV
+Qo/w+y3OvpsuOQUTzZ4G5TXoSMoAWLRtunbhuUSYZTc8ET/n8Z3PMktl73XqXmBE
+3J8JzmkgxkffPF7SaVX8xSeXfcZgT3U5sEXI8Sl5r4YbNbsEmL3pgTl8tfDLd4vJ
+dXAQXbLwwHv36czTsV3iHf5IEZc25GNV0FPIwsF/BBMBAgApAhsDBwsJCAcDAgEG
+FQgCCQoLBBYCAwECHgECF4AFAlhytLQFCQ9brdAACgkQWuQgzAIJmJ4huw//Qt3j
+P/2V+zuHuLT3YBUKgPAl9vOnSEpze+C6tOOdT8cOTKWybwvLBA3AYNxHRp4ERelv
+BkUJzB2Soaq1WZ/y283BNW/gsco5gk+MV1jiddL1IGiZCTUBoN1S5eCJoVoEV9eo
+1/qaCmvOTbG1VptM5dlPWcTn0CIfkwkIs47uy+nyupJ2Bt68OdWFi1iduvCoMbBo
+KLzFReHv4Or2hVcGQpoURC/Z6vP7Q3DCZ58LbbAHVh0+WsUSTh67t6K6Grco50Mo
+LKG6DHvUnCgvlWUrdu5en7Bem33LerKiGoVtVIVfzGo1hfz43XEAYPgVBP4zM5og
+dpqHZdbGr69PzuMI750R0I5oeVNlVcCeGDBlaMuk07qxC9DzompcyDKi4axrar9W
+yHn5O3eBQKhoPe4F9R7ATiiloZl83QTIEdLnipcI5W2BBE8mvV70NoA0EvBWtBvk
+W213rCSwoZdOzt9ro385FJcyZ2hkkvJd4Fd9vIO/j0+YrvWQUJS5fl/hBoQ4YW/6
+kSgX9HlByBZg71QUPxU1t0pHirkeMIEOs/En63ckvw3PckCjc+8F7FBM2IS1eXjX
+Qa2xXFDh6l33Py2MzFxii+/HzVBbmS8bO5cPKX2+nTQWfA6rIGzJgl4m6DSPidSF
+Ei4Wl2DJVu5i6ate/lPAaX2P0FFczYDd5a5yR8jCwX8EEwECACkCGwMHCwkIBwMC
+AQYVCAIJCgsEFgIDAQIeAQIXgAUCV8FbCgUJDMkgpAAKCRBa5CDMAgmYnsPEEADI
+bj0VxbqTEJvLIG2HTslgs8V/bQsC6ySjDdcIihJ2WIfy9P2xX7DZXZ+F17HYPRfE
+SuTN3BJ3CDq8OsX5Ox5JXc+jLvoL/V307UpyfsByUZvDYUD49yVDnNdvxrPhxlXv
+l95Hs68hHh3ZImVkHxOsE/I8N9lBr4l8cwsjfuanab6rIIIwesL+UEYdlMKQ8phR
+ADvskRTxS3M79lGZo1bt4SYqPFrcyapkQO5h/lPTE73BgJWN7fNrMSkR7Xh5Nfsa
+PzzufD6CqbCKicQ7tdRssaIXPSEJaCMyDdlgyjxX0RNOjzOdSjWC0AeXBR6qzoNP
+vMunif8q3l6eElDiJoEKPJ+DvLapdTnm0OWGgA9Dv16BnCiZudv1g/jPDBV7EB4F
+R6phtPoorTGwpv/3BGCQjWIUxqLvYv4MNLd77PgXVaf080GhydelQGl63RDq9Uxu
+xu1mWSapM+Z534IsDLO91qoAJoLjCbN/vKLGDclmpI/yeq8GQccjoEQo/CFUr3fg
+OBEHiO2LJ4+BmqE7UgSeThItBMrKd4/0KYB5lgWtnm2iXBdWf73XZlUBMRbCZ0zE
+sEyrD8P1PH+5xt/E6j5TcCmc4RHz04sM3CdGQzHcd7qiRZeJXOVChwxC96bgJx+4
+/cpHt3NHOGbKvpgiFF11U6ki1VHyc5vBseINCGIW+8LBfwQTAQIAKQUCUneP+QIb
+AwUJCJ2ibQcLCQgHAwIBBhUIAgkKCwQWAgMBAh4BAheAAAoJEFrkIMwCCZieOcQP
+/R0EfF7DgshCWfgn3puq10kP69RLLVWPcHDMp76Wh9S7xqexbFsIhmblL0B/PvoR
+aq4Lm+7iPUNlER9w5R4xzAUPVzytPKxF7SOBga8lDI0MHaint68IrmyShp3RY/zZ
+zHhjXj8A1WyI3jtj/QkGFfFpQrRjfZ7IAAVCGcWW9XJFe6w88IP08Fuk3HMQ6oMS
+oJTr1gfaEdtYvXOPFi/oscSAeAavibRY2dbqTYlq8AU8o6IotFCYJdlWqaeYBFDW
+h3p528nD9tjUTeg8GTMl2U7/963bnEI9gRxCL+wOg1oR/ARlYYCq05PLpn2NuOTP
+wfoUhXyH+t52dX8OuGXc6+TLeNsblH0aOaUmF96cWncEk3A+jkxsDafgA3dkZp9O
+jEOrqUHrW+WnL8Il8x2OHP+lw7EnBuw71Tf5ckAqGHzXh0K3jh6P3QiwGAZoKXFI
+05HV+Nvbg/EfV0pFu45tq2UJFsh/3iBi8waxaLXO2JzURufw9ioGPYbGnJ3gfuHH
+9KihsrBln7YepWTFVJGrlkvg8JbrasxpbkW+rlyFBgLny/XFH4r8LrTPgVSz8Av2
+GTwevzARpk/qE+6lE3zQaWw3DSc8+0aIV3BcxH5qC+EyjDkAkHmTL0LOxU5ATGQg
+4am+ELvNAmrk9WyuXPxOhLBCS37ddlkBEzDNef+6tlX5wsBcBBABCAAGBQJWBMwS
+AAoJEFMepnUjlhNk/4sH/3OlOKoS2dQxjpzKcc9FF52pLQO0wgSiOybize9U1SxH
+stQMRPRQNuImOgOemK02PFZ756l2ZKEU1ISoQ9ekJ9+KEVFWuAyJko6wVHjDZ3ZV
+3O/anka6Wj7uU7ERF8T2jkum80mkQwg08PkMxRBgQWajGxS6GABv9NJ5Pk0G21h3
+/0hfPEAHpTSlkBVnLtUjdnUN4+WnC6CFn059A1HhhgpPm+1rrpn4znylhAsIHSKk
+rJkbOphRHIbhJHHVg7WgDmV9OEXRW/nSr1ADuhiOEUphKUwVmb7/i71bwNCBaCSl
+TFuBIQGj5ZqMPrdiI4UOed+9a7pm0Y0e3LmmiP5qiofNK0FsZXhleSBTb2tvbG92
+IDxhbGV4ZXkrZ2VudG9vQGFzb2tvbG92Lm9yZz7CwZQEEwEIAD4CGwMFCwkIBwIG
+FQoJCAsCBBYCAwECHgECF4AWIQTVgjystHcZHKwAdVVa5CDMAgmYngUCZ7UeCgUJ
+ImB+KgAKCRBa5CDMAgmYnuXsD/4gypFzR3RKyBxbViy/vw58sKUbUx/a9+EsEMxN
+aVgwvgb49R4I3KEYl0xOLxYOY2+1gmvmdIRKH8inAronhxMRNegH2ckpqNHZ7tUQ
+H85pozg7IVHhtkAqiF4p/uVLXl+dVQGu3izYtKM9+8zX/hx0Xe1FcFZhAtdhTfn3
+y6y64k8CkXU9Yy3png7Jr+G00QLgMCUEsTC8tcQukTFw659DO3slBLhG/5CloTJF
+zIUk0d0Ci+efmjuu+nJLq6vb2+wx0QUAr58SgI7hhx/aTESm46ElIZqmwfZrKPDi
+QB55y6l70PqDDds2d4cfc5tt0uqqniXbkgPeZkyEl+EwtSA2De81PPVyp/7c/EoF
+cbE7gufyUyJvZbyMtmLZNSRmlOU5Ns25VmZX7XLFlU9dRg1/PUMk4f5bmh/uuMb5
+FZq4MMjcUY+qXp4e8OuD65/E70h1LBQx4x5gdl/6f0Yp43anFsGM3uK2ZJh/B8I8
+JQlwsgqnyAR16uQtUnOKePlPfRgOhEke7Nsd3EX8X1N/nkXQebI69AH9wbb6zPOz
+dLQ4QpC83sGfmjUqag3B+ARa+GFQm29O0DqbNaICo6Q9o+nH3iVygvFlLvXbev/c
+mLcUsd/h/JVekdVXGtIsIpJfSwqpWLNW2jOaxlLBTq/tgJptLhvsuAW3UgdOsQEu
+XCSpQsLBlAQTAQgAPgIbAwULCQgHAgYVCgkICwIEFgIDAQIeAQIXgBYhBNWCPKy0
+dxkcrAB1VVrkIMwCCZieBQJjp5RkBQkakI2AAAoJEFrkIMwCCZieUSoQAJ+FzcDF
+WdZHzuxQfM+pu0SYbwuKV+FLbjYLhCj6c+iDkgGqxoUvcQv47/3N9MchLeF9q9C5
+ofs8Z/ojeFrDFmq6bRz1Nb4BsbEyssj9Zym7oDyPq1PeEsH4btCs1VEuLuNgvqu6
+qKeGx5TgCRNZx5B+AayAMHdTtIsOk0a9ih4pNg8OkNTDybMxHFwykqEL5IV7sMMC
+z+E7JZS84DCsY1OrWP3TAmPItZNhP1SW/DWZOOiMfkrxZ3xzlsVm8HVKCUD9kdqw
+CaBkAxm0vyb23W5tnXvahVf+ohoJjIDOGE8o1ue6SC3WHz7i1vtgzGXQsTWxHK7w
+gyiznTy535Gko9Wwa3euh6NlfewKzaY2xtmNQJVTb4VuuVgf4wFDeuKeGIiZP7XW
+LZae3ZRdCyHygsfAwHXQ8sxiEMsgVvIJjm4PW2jIRwCEIL2aCfpjtdWR+J76enqd
+H8TXWdacoHQ76zlvkshipB6DLYURtHAOEqzCWLClh6G7gMtl/jTqDXLaDg78j8Wn
+ktwZzL1gWtsvTzOa0K/ZRFQga+12o9hVWXibl035r94g2EXMs29vSJees395C6n0
+PjnHtmgOnD3lOdQgAz18AxTT+vhj+UKjPUKnxk+ogQP9cx5LhppUDvxI3COgZxD3
+UPZW9cZKHF/7jmBGMDqMgp2qw6y7Iyh7pjnRwsGUBBMBCAA+FiEE1YI8rLR3GRys
+AHVVWuQgzAIJmJ4FAl7KfqACGwMFCRTpAigFCwkIBwIGFQoJCAsCBBYCAwECHgEC
+F4AACgkQWuQgzAIJmJ6KShAAgnyTsYMtS8IECVn8e72s4XIvwHlJ7fZS9Mxm9pde
+FKQ52OQ6JxD+WhrfekXhlaeAX+mm2+rvhBjtQqauFaA/4JJK8kQ2MPMpAO4ESxrf
+bu+PKMk/cJotGWtH1xqOAbUo5tOTRbp1L5UYuSuahw0F2Cpx41ijUuWz0Fk/SVXd
+raPhZPIxDRpGmXGC+d+yCb5bnJIhAxjR1rOg19m+d1X3cefOkh/MtuK+Ia9bHDef
+3IB2Ka7nqeGcjDMM8/iLN4N7sR22CQcfxThe0Xu8DSTLe19mm2S0c2wGOQU6PgZh
+LP27YVCfAqciDjdiKsnLa+oUadrycY4iR7/5+ol8Cm7/FqminqmGKpAVfRxQ0PoR
+DBO7gbqOtJe10RwFGq3JfEFHfaxKWa+1UtftF4z/wkAoyFqlRhJJn1Bs4UAQIjVF
+2Nm6+tz7jBuYny+ItPzvThujy5gh9zmvrBXFRFDY4Y4lQckwKmVRpAAApcFoZnaM
+TksVNNHNXVDxTxUjK9E025f0YpJ/EUyl2O1gzsaQZbwyphfgR3tTXuUnmVLe84TY
+MfK7kKOKToGROkDsFAq6ugYCZcA2JHmASqZxnAf6iHmzyPVK+nCAOFIFL+BECHQ/
+/3NxkoKlAhfyhipEqbJaK/mu6LpW2vhTbYEv4MslM0FM+9SV6vpAQFglNEC/joVh
+mNXNK0FsZXhleSBTb2tvbG92IDxhbGV4ZXlAYWxleGV5c29rb2xvdi5jby5jYz7C
+wV8EMAEIAAkFAlZoQfoCHSAACgkQWuQgzAIJmJ5F0A//VLE/Eg2RzIA1u6bj2F/+
+q3vVxZGh27druc70/lyERHvFzuEpwedWJY9hn8QW8hfoSEf+KF5FR7DQ31l5072e
+n6G4k8uNrVhwWXdraNMV+FoFjiFbUPhWu09tHiyqtuPrxVnibPXWqZ66/gta1gbc
++5jxm0MA+13czka3DOm26/Vg2IIt6GCGM4nUaBSC2iifbVH/FbxHc7KlzK+itdqi
+gp5Ij2qg1tRY/W6t5zFuiiaxYE6ni9sUljRz/nO+rk5I/ZcTy69rtVmy0z3we58j
+sxkzDs1SSJ6Qw3gjcMdsmlluFFGMm3X6P1X23pjJ7aPzVMkmD5BQEKKGm/Od184i
+/VGm6a2tai4eHvIn8/VBSNu2PqNpCXmL0GnO2d869hXqtVd5EqegaHVEaSLnyqKn
+xU6Fgnf33epvhkfJ6mdFmnHF7RFr/VuGnWRHnkT9ZtGUazBAywv041tIqUMcw22E
+R2exRei5n/cS9FOfqM7EqfO4BKm8RAjtJUzHB7ha/MRj6hQLsBf8mpNZJWKLhOzu
+vqveTljuGaD6xfTvxZ8zxCROdZrer5VvQkkV35V+e4nJGRAy4Mn0g38C8VF+Bbvz
+Ia0Noes3ZJ36Nt/Xuvy+wL4EfmBvxLtEs/DFAiTCwX9NAujjDVXlTA81A/Ig7/vp
+D+g+F2EN9MjtUBTdPod6GyrCwX4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQWAgMB
+Ah4BAheABQJRtKlUBQkInaJtAAoJEFrkIMwCCZiehhwP/1fpWUGRKqsXbJQZwQkd
+OvtQm06p/P0QW6/9PruCG0U32tku5oNNJBHbqg++S4T2TrDH/LnyQ6FI8Ykhszcp
+9WL0djNNlhVb1cKml1RNHMDIkh1WWZt5vpqoJJSNWwrmflMl6a/NBb4tyURUqfig
+9MM+OmL4QYHexaKKOmy++ICKJ31einwBdBiwlV9IJcTuivBvNkiaV6hkyPmFU3wS
+NiwFE7g05UUjqqSO78gATApbJbTORoBmf/B3hglF2JBvENLLtzBYRP652ZQj3NlI
+Py4skQEZUPJAxs16Bxvkl/WXX8baCR/SaRSIRXtBhALz91PRlSuhnKHXicfXDrmi
+N/QcG37U5mkAz98NAwnOYlGWxZwoSTeksNuYphf/xdIXvWy+jel5eIyuwWvk0sRs
+BLJ5eMyHjI/5VSzpA7SDWjNgBMIzGnTI8aeGc5RjFErGrddT7ZjMDJ53hBMH1WEI
+Toljuy7VDa5NvpKH4lOtUQglqV37YrG7gg8G/cbFFnBgA+vkPV8KuDalfAAHWJLB
+xPHOkSumPJVknYCBegjGsxbCfur43gg1CzABRQ+ZmYymHnHHqbCqt+rt783k4N6p
+8ZFmfacfJo5iHFk23RyAK5EocQIqlcbUhbE17Ai7iBr7j+KCaODJnCcxe40C1K+E
+7M7GlZehS1Ihj+uSagFUPTI3wsF+BBMBAgAoBQJOuqFgAhsDBQkDwmcABgsJCAcD
+AgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBa5CDMAgmYntt6D/90ZJ+CymIoYxtTLWUQ
+qKDY7DCiwQ5UaMW/LHD8Q4U/fRk5AlAasnzV+HN7R138z0m+DeNjuNHOOphiWRYj
+V8HwWEqrpYbi2nirUdrR6sA3ZBX4ud2kLf0g/p3HPu/GSf9rOtiywfL59ubIRK2W
+jTeFAMvDcfUr36wPNSL+S6sSYWb8zG6G/hog4IpwAThXyx7oRPMw9qStIrUl4JZT
+hEe+wMwEIgYZHemZIwr1tinRKcw7QHtkkv9SrDjcWNupKX56Ha3tV1fyHFDLNXKn
+MVDLVREJqGiOmUWUDYzgxzMPM2mSemRPd4W+kZpav+uA3ZaNty3s0Z9Oh4qn9OFS
+98bVuH32IVD038Yu/dthvVqERL9XuS36sNUeIb/BtrsAnUhmTlfM0ylVKYcVFQBO
+63lQwRWMqz5oKHi1ZIrmE0AmK9inbJjGJ0pKa+HJUlCF2a3+53ttVgFb3e5YBUsF
+L+DM6H6cqqxqQ1/pTYl4oPruEuWHlKY20qbE47o0qMqqCCjAHVumvv0TvI59OEMr
+vJXLyKB/IX8+M32wlCMAWawJSGMWUoJl0uwcUuvLsyxwVsLWYHIu1s21PJZDLFA0
+9cNSoMTPoOBCWpJeTmzgz3pmuD527S7IKiAvw5GdM31uEA1SaL4R8pO3A7pumtQD
+ftO/s+nrbEcaUw751oMU76rfb8LAXAQQAQgABgUCVgTMEgAKCRBTHqZ1I5YTZLEn
+B/0VfcsDOf+wkNXFPo0TW3mleY9A67OSICyJ5CuAT0d1TLHTcRG6uSnp4BTdmHSM
+w35jPUM1nvK9jG2jSfRGBrXEhddNddcnsHwSt+0FO8fPwZnPET4N3TZHik9HBoKh
+rrCvcAlLKCfIgKCRqdjTcnid7tfSer2mudGPy6wWJRhAiDznZ3eVPLxwPsEO1sZA
+uLN/0yCu98lvdZQ4xh5ic0DkfCMKlOGZ9lTtt4W0ds4XvLHzcxBrN+HrsIEC0DOU
+r0DwUSw8fFiJMjHh+qMpJmTrXzi5yuZbH36ZGXHLffLf/98cRsRRMGY/JCx/Rol/
+LkH8VHI3qIPVDMn8SXI6xb1gzSRBbGV4ZXkgU29rb2xvdiA8YWxleGV5QGFzb2tv
+bG92Lm9yZz7CwZgEEwEIAEICGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAhkB
+FiEE1YI8rLR3GRysAHVVWuQgzAIJmJ4FAme1HgoFCSJgfioACgkQWuQgzAIJmJ6b
+yBAAirx+gC91vQjCXoUJ81rzEb2VDJLSOoo4LV5iqFFwPyFmB+RlQa7xbijVqG8H
+tUlOgjLfZ4RCtxHgcE408TIy4YWP9xOSgM2auKKb3EAgC69a5xU9UWw9MZmJwqrP
+GNaY5TFpoNyzbAGqfAcBmrCLjdMGOGRcchtg7QKl5BrsNkQbMcD7au622NgCbpHv
+pbTWuEBRNCkQnnus8syp8T+FiXbEgx0zrGq85bo17+nP6hEFew8f9oqWK2dpJbAE
+mIvpymD45wmWUOevM6POaJ3g4Vl3jhiSkGlELXuphznC4WNDiuEFb4qSfDKjmlAm
+wDjHWiNZRFL8aPzeS4QAIiCzeyoIplRWwZlTufRSk4ZKzVJMO/fEyGAukdrEZlCm
+04Z7w0+uBG/e0Q/X/0xqBdqZo9OJLze1HjZow9hI/J3DL+gbkZ2N3z6jcsq4YZoT
+pD0ia+rwoYC+QET7+hRkIh6ZK7UgR1muirGgWByX8TvFciI8joyo9y8wgfkymw4V
+T1ThKQRlsWFWKjpLPWoCF8pw1titc2nJb6yr9jRjbsEW3eFbmLxPOdztKBF6uD/n
+F5AOnRlLtd6DvNumQiNDdrnZba0AH5fdAzHHXQr+xk0DxEhAny7wfy7S/zR6BdWo
+2/21ejoDGoZXdtmc5yWq4sOTr5f5CHTzxzfmFkfIZb0HOuXCwZgEEwEIAEICGwMG
+CwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAhkBFiEE1YI8rLR3GRysAHVVWuQgzAIJ
+mJ4FAmOnlGAFCRqQjYAACgkQWuQgzAIJmJ682g//VjFXKfxtEWVUMdAn55MEdU9N
+4qIGn4jXh3NcuyeN5i4XVMagVZlfEo2aUpsTWR+Xeeqq+heK0QNO05nA9U8XnPSN
+p0GrTgPPmlg2uGQtz16q9X27/NekN5CZAX+RBWKPB9AZlfPxh7nNN7RPUVnp32k2
+BB0PEYWTAv6otgu7enR3TV/jE4urOwc+Umo3uuvuOzHGxY/lDJW4opSVE+dHGJld
+NwhhjKXVuSg9ldzTydhZnlorrB2oHwlRGqaPajYFICJgGLxv6YhEhWtvvJIoE7al
+Cxa8mTRg2QEah9yXPRFfj4MtzNp8ZtCQD4WCQ0/GJyf7ZgKqG2RmpgHfWYk5HhJo
+lt7QKKfje0ddvBb+rLZG4++iDbuCQ4175TTEtoQxJn2O8iCv8zbIYUL3p5Ok2MCu
+H41eunvzpaOY4knyc8sr8lnUaGqcHvj3ggjDE9Rnc4Z822tBkR0S3xrtcJIMBALg
+z5NcZVIpJMUP1DADphsohTqGvTnomoyNat/SQaQs2QYZk6okXtrDYCcmsZQlQmAw
+NkLjKkrzYXdVsVlsAV0UJ1A93buTcqidcKzm/T2H7+8ZNPYtInKmiF1jdsoJIRMG
+JzQTK59NVRvoFk06yeJnbwE4EcO0bfw6lmFUhbPyxpbkndMn5JozxLCdd8lYI7Yd
+lCPdFa7VMHyBEr24DZLCwZgEEwEIAEICGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4B
+AheAAhkBFiEE1YI8rLR3GRysAHVVWuQgzAIJmJ4FAl4ACQgFCRTpAigACgkQWuQg
+zAIJmJ7fQA/+OE2WGnMZi7pWWMHI3SplG6FP9yY/oFu5ejY5JpFRfTfqSXkiHQKh
+4jH/DsylGjIOc2wPRKhbUNrDviYGyVzHEewn3xA8S0qwJYyKLyuQYH+gvObioHY4
+VbfkaulQK8CBAtvXwLSU/tKsvBJuY8Ltd9h7M75KHq5XGWUTgd8SgE8rRApAn4fv
+IH+FyLnQWM/Q1A8Vs6MIU0ixLwhkD2ZudcPm8CrOPd2V4KLN1UOY7hAx+d0O4V4v
+Y4W7mTRZxh1hZ3oalDjcVTglc1i7SSyOUbs38Z47hPZ0dXBi/hszFmceHkqtlxmz
+/ukIPiF6wExB7Ksup1Rnck2viFAGCvfxqIyfib6QzBrF4CGmkgLwAI6iuHbHQohQ
+ditPaf9J4l7LO+kW7DR0CMps8tjsu6+Y6uXw/L4tOwxyYUrFnlQ6HfhvGuHx4wQb
++/EBKApXwgoQw0Zv8uW9ARpRyVuvPLYmimJxUDsgwShSUd4MXHmTTiUqUOOyhluC
+vyO9Xqk0aV+1Sd8iiJ946LMvsmfE4Lx2hUSC1+o1BokoVWUr5WXEfJggYCi8sv4M
+UFdgbFaBW+MW7gEhm8hrDnQPgHLLkb8JRxofpHUyjca90LF+QSpvHiGcaGgCm2WU
+mWX1EDAj0fCxyVGMWcAwkgoLBJvE+1bpnqY6bRHLifb1QClPVY4ATunCwYEEEwEC
+ACsCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheAAhkBBQJYcrSwBQkPW63QAAoJ
+EFrkIMwCCZie15oP/RGXscYqFlG6TTAVG1zC4z1kF8clfv+d5wxp5fFiVrz0rSG7
+m7pa2rV0SbS/CfAtbbkzIEj/OW0bJ1Mb9REzsIO3rBE9dy0d7Kgw1iM76hYorqTQ
+YdwiOeFG++tSbGQQpK7Yio4OhA8Dit4QgmkLd0XQE3iwBf0MaJPFgBxIFoQndLI0
+pYTMqk4jLxaaHeu/AFmKDoz2291XPqXZe6gtIQWiMCoI51t3qMI6Elg4PFclEm9b
+7cF8pJyh2dWZszo48U0RNUcOvTUDMyQhDGWhdS+SXaJUG0uHZtuzUTdRXeYYF4X3
+u3aKvxS3f3/LIDbyC1HLXc6SNxzZjeUFx9sWg5GN3xXNnvvKhyusDzl9iCjAUwV8
+J1xcSSEHA1xCJlaPFNbGbYgAWp9mH3nNJp1Tv6H4mK0e8+YaxZ1H7k1LN11FlJiU
+qsz5n+N820W053YZS5y3R4kc9+fnBYZXodZg4BWxUUArQO2uzRdAiHAFmcRXulrX
+E2/NyeA6vurbDtNYUvW956WAI1eqQdp0CQJQx+GvWDmQf32P/y3NNJLfFJ01dTRc
+N7rFAWkqucCIAdyL0kQN/BR3fsh9OUe/Z72VJI7ch4hPc51G3oQSCY7Qf/wN3ugY
+GvYvdXm0AWjfzC2vHM2zoz+uNXC3uNf1hA7B3vqj+KDbp4yd2xwb8ORftEWFwsGB
+BBMBAgArAhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgAIZAQUCV8FbBAUJDMkg
+pAAKCRBa5CDMAgmYnrbpEACxYAsYcp/yghaCrMiRwhEl4Kcy2rDOtAmNEU0xfft6
+Ua6EXqrc69isiPdz+E1z/VKr1ZTmd2dNPGZFu2BditYiaGajX3xT17OKJw78SVGF
+OY4UOlrwrlHXOpzLp8K/hPXbNvZivsiBw6Cl1ipyIH6r/5hUmynBZM2PgF9pu8Mv
+dFD4kShho6NcxNvIbomUWlWJ8KBy4X9vzAN1pVJvPHHjia5Kh+mrCKopfazDXTCg
+NE2ysYNvhQ25VNR/1xfcIW4HRaC9Jse2EgrvDSC0J3x7lFm+OkMcjvDqPrnhSCwt
+y8DQPntvOTX4O6wU4NwjY0Y7PojqSZn2UcUL61YFhLDvKuicjXIaXhPYn/133kO+
+YQztsLBUabpK+x1LtJV7mcG8NG16dBqQ261Lr2um+MX4tjl+mre4P9663bNFXNJm
+mMw5FJUeEBqnrfVtp+ZYlG6ESWwWeRtkDm2vzy/OfMuXlXvcYWvT9iUB5+BJkzIX
+O0Iy3kTZ8Jv+eUYZz08VDBpfLqq0gkKkHhTa8Yyk8oAag6WdfZfwzCRnszmZmVr/
+wSt1O0Nbb1oFIFffcf6RAxiF8SBztcWaCQha7O2NaUK/1nza4AgGVO/vRfnSbfEt
+mZausFCS7Um+tMvXqaIMoZEMxFQ+Ar5MNQcfTCwOWnJnO1P9SeVdZMrq3cH/jRMl
+ScLBgQQTAQIAKwIbAwYLCQgHAwIGFQgCCQoLBBYCAwECHgECF4AFCQidom0FAlJ3
+j6YCGQEACgkQWuQgzAIJmJ5BxQ//f13UdjqvieBd25pbWszfhL5Jm2RAy+S2sazM
+rWiBxzWFk+gSASvczjGf2xnAcFY53Uy/EypEko6TQLq8NDmqKuE9d8p+Cie5+uLP
+p3EdPOTbS4D9u2jqBIYEcs/naUBczud1mCmYRmvjZFs7Wsg8Dd4ONUF4pw1abmG4
+7Wtp9323CTHSDS1AGFEXV9/uV720Tf34tq1zr9gDXBelg9tuA9yLaMuOctOJNlYO
+MvjT2X/GVHda1co49oDqgV4EgLYe/eB6W+N4YgRLFtb1ELNwTQ5bF06WHsvuNhmM
+nq2qe1ttIp314X02zwxAPlh3SXjzBspF6skjwP2ASysO4V/p+UiDaU4a6NDkgEmS
+N+cqoL03K3Cn6OEPKbWEj6hAQmYLybu3HE/zCZDcdzlmnYv2wmsIHJZ4lXCEiq9B
+2VgOgEAqYSB6cmAfDV9SWzqC46EOzueyAC+LKRe8fOCEWUHr2gU4qUE+Jy8ddmbJ
+q9KLYkQAnNdS/nPCwpot135iRo0I6ViNd1SG/wfoxsJaVe7f5EgN7xP0Vkpd2kxA
+GgP42VFR9N0Ex0sd1BkWrMs46lemeTkV9w/OV7LEeIXhtpAbyxsemdAUI8t0n6/Y
+fyQk2RlHgrE9Q9UtppxEF3DLhvMNEm1ayCxlA4ZYE6sz1G+zTWHLJUd636oxekcK
+ftzWbI3CwX4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQWAgMBAh4BAheABQJRtKlN
+BQkInaJtAAoJEFrkIMwCCZieXzEP/jiHoepRv7/C0TnQRcnWe+mvEt/6P8Wf4fe6
+XPTypaOv3QCGXa0nPuuS7ek9HIrzluhgji4ucDJakua/BhgZ3U14MCyTLmKShldk
+x33vjXxwOkCfI0jJarwdH9VJxZz+XLCj/i4i3GSoXKp+dq1j+OBHJ4Wb8nS/SD1E
+P8MLcyQ8I9X3JYyiqNjHKRIeQFPBmtrIX12wUi0mae1xRH1Kkpm28fLBsgggsBIo
+NbCGDyVwXHn/1qUCL8RY5yBCer+riB/RQ1qdWun76H8AkflvVSpJVfxXvACQI/JD
+bUomwrUa3mzMHVZLQKpdjrpz6Sp/FZtn6R+1jUI7q+5YWoNydg0unCtduettp1e4
+Dss+9u3mOTQ9s+P7nYU5bcPfJC8jvBSf1F0UPSSxdACMZRn0BEJwC3MgQEcOrGls
+ng69EdOnJf4MBamYS1LBUhecmQVP4025b1c9x7habR5KHiC1rLEp5JxocZdoM5G8
+Guk8RPoUIz3so7ImPCldFsg7cDGhWPJkZAShlCyTXMBH4sHggAV9apAAf41b5qdp
+EhVcniXQfYf8CQhvjc9ZXLmvKzOT2Bw0G/YWi9KOAL+3hO6uP+msWje+pR3LAvon
+lkPJsCmq1rdOo9AnKVAwP8jl7QX9U3lIZ3h0SnXivBtAuJv8oaYH369/UFA+rye1
+I9SjsRMmwsBcBBABCAAGBQJWBMwSAAoJEFMepnUjlhNkNMIIAJCVlnE/HZP47M7e
+5TK/6x+tws0Te1C/41yV4zemVg1x8tiFVkO4867fblQI/DdHJtSD+t9yWuwYjJKl
+yQZkfk+Yyl6nROx4hhs+m6aPC1cmwZN0pzlphZ5su9OVv9xTMvvOPGqHY50M5dHS
+OczQoXMFptQIDqToZE2P7D+NbvShXmgZZOrbPbc7di7Gs/mDu1sj2WDj2r6FecON
+NiigEvdbRrOkhp9T8/bBgtTGz8iyWlTulevQ/OC4WIKJqC0zCVu8xpbf0iVWJbfx
+58B2x53jHfc0Xavirv0OzXD/E7CbLhXGnCJfCjwnyACu/7r2OmxpkbsdUOaHg4RE
+DY1h50vNI0FsZXhleSBTb2tvbG92IDxrdG9uaWJ1ZEBnbWFpbC5jb20+wsGVBBMB
+CAA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBNWCPKy0dxkcrAB1VVrk
+IMwCCZieBQJntR4KBQkiYH4qAAoJEFrkIMwCCZiemz8QALRHxpoyOAjKdfZ65o+K
+w8GcMsOWZvwjm4MRjbOPuU4UoFNV3C3377aHOq/CnE96bF/J9e20oSJzHw9/R2EZ
+UBYBMQ87y5yStfSzEJhE89ej47cnQ5yp16k0fZx9BWXmN65lh30Uu/WwRZNav02u
+WB6nhbA2QxFzmU6pEN/0U14THLOQiyyhwtb0kXw0EBbsYL9t3DTSMUu+uZkrHvNH
+qdgEz2ix+mxfvRTr5yC49NMzLpQnUqCYybKWupR3ZCVJnqcSNlJCf+S9PUtusvaJ
+VKUHxB65wT7mhXCp7JR3N6cACfUUjyXAj2Ajj4qNgdxZR2sQMjG6IFwPWCowLbiE
+RfjgXIowtJqPmIhKTZDO17oqNUFgxYR7ZnXe9ROQhnkex9CywAguf/HkjSfA3OGt
+cUpiXWkppaoEayE5hOKHw1QXSV12bZJEC5nINS0MivFQNsYdo2k5+XtiF66wgkxJ
+e15z9dxopu7dYNBzuAuoB8OdIqu5WQrR473NTYKob9LAiKlVqD84HqNJpCC3sG+4
+v8C+5nYvipsjK1kIfNnWu9k1zzaY9umdrg5g2QuQswlgoYwdAbudAZ30Xq7m8hGw
+OnlINMklqmOUTbrCPgxdbblBuJ2fnbJEFkH03JPFGUITAbMt+Kj2yjSIyPwXyF6v
+MCHwdkT1fr684I6IG369dxQOwsGVBBMBCAA/AhsDBgsJCAcDAgYVCAIJCgsEFgID
+AQIeAQIXgBYhBNWCPKy0dxkcrAB1VVrkIMwCCZieBQJjp5RkBQkakI2AAAoJEFrk
+IMwCCZieGQgP+wcQrLlmmxqhovM4fXylVqyYwGyPis6tO/DvSyzS5LG8Z8kTIpZ3
+HWp+dYWe41lGNfE3kfor6YCvJ8fBsLho2DjEediffodQWYrtplgsltCtkSCHfTQQ
+mgo7hjHymvuH5YUnp8atuB8Sk/EHtLL1FBH+HhffxjURAhhcI7rOIP6S4p3aw2C7
+acAHNKF4rRqp7eEXFD0QlZlNWCO9bPf+XF+8eAnCjLeLHb1B+yo09DQsHBlxtbZS
+cZ9ySOxZxRUOeHbPwWExBCLnqp//VfdHAqpmox0fhpoBP0g77bPwtV/T2n0Yt7+V
+zL+wKNn8YxoNP0J4NimnrEHjil0QHUMQdCLxDYpEoz7FdSroPl/EbiFicF+8XhLh
+j+ITRvOka4m45A+KgrojHf0ZV5sm7qgF3obJHzPsL5kC+eRA1nveHrJHa85kPEHL
+dX86PJAS+VxGkoHncyzYSXt3UMDCbGF4IEZWrXAsOD33+E0vJMqVLQi+5QG4uN1T
+FEwIxDMT8GsHlO+GFZQttejJpB5ABvwcn+o6JVGmCnQvXIGu4b0t68NzN5UMtqt5
+S4+UDjsvBcw8OdC4G8rWP/mOhqR3SMqVZPyi5Y5a8TboSqqJkY42QlegkREoARLM
+uGOiIg/7J3t6kgbDsrImYd/Crk8YnBihygfL2AJvwUJ0B9aDIhtmtmDqwsGVBBMB
+CAA/AhsDBgsJCAcDAgYVCAIJCgsEFgIDAQIeAQIXgBYhBNWCPKy0dxkcrAB1VVrk
+IMwCCZieBQJeAAkOBQkU6QIoAAoJEFrkIMwCCZieH1kP/3ahkkk5IjAzAb0xybyG
+jHVEw+uwfMtrCM93IMG9kprkIlOM6P+Cyhs/ETXB122AC9iiVay4IJBuuKXDexYu
+KRfBEW2Hksg8Uby/1VKcW74YLfRWoiWcdNnbtBJHTdpMIYYVDs8uDQtxDeg6oxfJ
+ALN0NIjgP5WS7Uo4QmTkdnoKN+BeGUe4Tehb0v6hmHH3GG1IlfK2jDdUTnio7zNn
+HCvxI6rSfPUZE66mt0MALEBqXZIHi0VdPny6TcW9euAAaOWCnZvVQsJSPLtKibBE
+2M4WebxMyg/2shvO0uIPvXrYDXurKwkNcQRRZB/k4wdHL/SjUFRmLVYkGMg07t50
+jm9bSTRoXUVLAT8YOkzMMLWFf++Mr6n2dHQ2QwyQGqWeuxmKudmNFpg3H1CYXi7B
+0aQPfjYNat1CnQkv0PrYpkgNienD0wFwMctEKkc5UJliGbyFtePaheEs67Dqd647
+2HHqNzbnX0gU3jv3VpeBwikYMOXzbR2GVv61lZEG8sIgKkPxUZYNmMy+kA/pKiCA
+f901uV3R2NlSVkaI+ZiyxCtCkQoggD8s8QJ92oDgzK/Xjc9Ms3/AZhQ4QoMoujaY
+V2FZ0AgB45SCgfl7w2aPC4lEJsLsgpZY6ZyTgF3v8BxYVe/BJ2d2LJ1FUdPcQMV0
+1MeX1a2gAaAVChEaqci0nigAwsF+BBMBAgAoAhsDBgsJCAcDAgYVCAIJCgsEFgID
+AQIeAQIXgAUCWHK0tAUJD1ut0AAKCRBa5CDMAgmYniRgD/sGZqT8E50c2hJAIbMC
+P6tOJY25NWZow9T1wisO+HZW+kswdl9WS1DukS+UpmHj/3dJcpj0ysWDZTlOEU74
+7nB3rZLR1OZRm7OHniq/FNCAIDVKGAK8OP7pgacjKtu2Qs4ISIVPX4ZHakF1pl+y
+I7W4ZnXvso5HtNKiH3lkDv965TSdZf7vkNO1osq9vAbT8f28XX8A0BWwYxATDZr8
+vkkNzRTFRgPDJ2QuijD4lXNLBiaFLm4uvn5tK/yLa69wU6IuS0Giy78yu3i86/Mx
+1SUDVO2jwT3jkJl1EVqFTsscecxFmx6rL/3bxLwvswu4KfIIgXUHq0l5yheCNoC3
+JEV/CGellvHmv15uxPqcctdJaYahAJPwBto/6m61BN5PR7HN/QljmQuiwpPRd2SU
+YfK46k6WJ36TIQldOAtkGSkFb0R7hC703NO9oEwS3xszt32mPnp7qL7JsxgA6b14
+zzA/ppQatKYyWbV3WTZfg+LDKsu6Lukn/ahGCpcMbAZcSLh0SPKtcw8COoVMolTN
+gbn4OSZSiEPhYEP5JjJmfF4uG7kAkOxCUED5xGfKW7LqiEKvY+n7fZshXNhnOXq5
+NP5Zwi5jt3x4Xxz/2W8bresIefXTE4Ln8Sf4IJduMI/aXRcExqgLURd5qSh6giyn
+S7I7MpXPZ5yIv7BVJnb4irdRLcLBfgQTAQIAKAIbAwYLCQgHAwIGFQgCCQoLBBYC
+AwECHgECF4AFAlfBWwoFCQzJIKQACgkQWuQgzAIJmJ7d3BAAz7mBRKs1knfZv2v0
+ur5AUpflZhvgEaGAGKvR5AYj+1vW1LlGVx9kTfXpKCsgtAKe0CS0m0EZWELAYHFg
+kY8fl7F3NzJ2hApIl0RI4akbxahidsAvrJffp3x9jPSfexS4ZSl8ccZoao2yeZk5
+Tb22tI2srGzzKLCQzdH51NXEPhMDYHxvzDexRacfMyVhNIgP7dcdAflz2OMSwHok
+Zfm6Gw2jkb7WmGCnVTtzC74QjfXP0qD2lJwYuJxuWWGsBN7Pqqx9hJSWm3EvVeZS
+BldxZ4rHv/YZ5X+vIhXfB7lA1b10pZak3XgwtcXI55Rmctn1/85cqWRbAUTn2iID
+zfTbIME/yEuTtjKfvN7oZKpSHarY2loN/F6f8RYTW54Z6Cjv3KlggJ0FL4sziyJi
+PsGdbR/DvRpxc6EeV3u6x0REeP9Jox+BLEe0tkZIT2NLYGNFxj/DxxWsuZOSoLTJ
+lC+BGxsXNnTTscIRLxks3KE+q5LXZuO769pQvq+usLq5eMiGqc4ssqmoQbL1gqu6
+YyZPYm0/kltRJ/P2smQAwSdtwfwpe63jOEWg2elf9iLF3cER2KwJnzysFjvX+JqE
+e36wioN/0/ygQ7XdX+YImiZ6Csiej27SqO/BSyAkEQpS+iJzAdzXWRR4LUgwSUB4
+68gR6GezA3UCqEGPJTmwCAPPMy/CwX4EEwECACgCGwMGCwkIBwMCBhUIAgkKCwQW
+AgMBAh4BAheABQJRtKlUBQkInaJtAAoJEFrkIMwCCZiee7kP/36Dt6yoUlv8KQ/x
+DK0wxlu5qUMxcGPNHg4CVMqUpUbZ0A8RFZoxbgD1ffojyNDVoKdY5L4cKlDkYGJ0
+oaqJVE19myJhRLnuXi65cwsZQNJllOYgYfg5VDKHJEGJZN0OIcz3xIgtSBoLDb2c
+tsRBg5Kcbl5LG3almBYmiWlhreKxa8W7KPITvIdOBX5WyJroXtEhMmGlHJqk1R4E
+CdMzNrbp4CuraMb+9JONB/V9TiR92eRuZ0YGKkfHmuZL0kgayNxRx2xSGs/5xX56
+/AlUwaLhVOK7bBwnqvag4O3J8/D18QEbdEX5FEws1v9zmreC/5ygEKxx/QX6fb9A
+Od6EfL7iUVcwoae1TkMB+4CBeZs6PKAE9dIMR0qPLg1PUC+ZjK6sverU1ZZTpsTk
+IhkPut1AYlNwrFM6XyzLO76N2ST4MX3U6sYS+aDehDDvoExOOyZRu2prILoecu9B
+S/zlkOzDrPkikDvBywwAU7ioxLzwKzPaLWgIyCU7I0dzVLhhxtMhLwLcO2PmkdMk
+kA4xOTI4cMFeb8hv0a5mpQsv3+0tIczScLCja2g5UoeAnXiYFsebGfDZIEckQnMv
+IVw7r1JjX/qFsZbSSxl1V+wBwO0/DISajLJ7rRNwCCOzVQlTbCdiVVhbpfzAb8SO
+AHda8mPHgoSHoWdMvdgDsM1OGddqwsF+BBMBAgAoBQJOuqjzAhsDBQkDwmcABgsJ
+CAcDAgYVCAIJCgsEFgIDAQIeAQIXgAAKCRBa5CDMAgmYnnwCEADFzncE7u6+67xk
+Zn7k4p24LerAL3tL+3zTKDDEdYsyzfrbSvNaeSn5moMwM4UJAt8i+b3kJ8h9AWXo
+KBkHFZy+HnV57cFyZTvn3sa/o82Y1vba7Ba8YbrpPUj6dHnX+ylDd/FfPOlFskAM
+bxLc5enKiUEhta0xJzmZFJAYTIXnQKbqnc+SCSWxbqoVLeFcfrEkfuGlVS9zKaYl
+JbUhZhsy9BbKBfjSu155RlMqJmm6NJ9HOAGLrsQlmrlRHDlRaZxXhdR3WqmHiGrE
+e2fryZD+9VeyenFy/oW+RPJaJAvaRgkLdyUm6AKZUzITvEch07knMvshNiOqPByY
+2n5dg+4Lop9b46Mu5iwxE+llkWX29jsL4qvFZXXOHnZeKRS8hJsul7qxyJwUdB+u
+Fuw7bwRzY0i+XiDeICEi1adXh1IQtQJN/gq4M9N+3tOJInTH07cTpisC1xKpGzdp
+/XmSGYL6ZAMlY+OgVOTt5t7VNlAomFH1p2TWaAFrbDEvfftmY+f9gTEVd8SGdoc1
+pwOFPRc6xjnc0H9w84rDd49yQCs0/uOSAWwq6ZwFOiMrVli1F37xttBq/JTo7GJD
+6HhT5C6DKF9s9zUcMhjPc479caABFHFwVTS/68voLcZyJ8VMrx9xL3LxD/eLvvxu
+rFRDZ8FSaew0clYmEq5E3UF9HjVbjsLAXAQQAQgABgUCVgTMEgAKCRBTHqZ1I5YT
+ZILcB/0UuuAJCYEbp3n90o/UA2QAX5BNqp6ZQZso0VWp0eI6VWLtFSmpUWtpNt7/
+U2tMCBDMPEj/FG5T5Rx+TveyFoYuRThtJgyyh6gOFQQ5PkEqM/ZljFiZYpXkJ107
+K096Ek3et4U3pSWUG3wntoHGc2C1jKiIp+5IOPqMrq8wecZR4gv+lgRX4ADpZ6pL
+HiOPhJTfy5Opwbc1xhQhBPFmg4JHSg7htbLjipSizRTYiac/QvnN8e/ZCRzI82kY
+pmHF0Ml/q5VVGcIWnkieTGMZru4tWMp8fcBQDUtlAKSp7jKCLRee0P56c73ehY7k
++UmGVGJZCsjh7zPwIV2X/Jw1LuUKzSNBbGV4ZXkgU29rb2xvdiA8c29rb2xvdkBn
+b29nbGUuY29tPsLBlAQTAQgAPgIbAwULCQgHAgYVCgkICwIEFgIDAQIeAQIXgBYh
+BNWCPKy0dxkcrAB1VVrkIMwCCZieBQJntR4KBQkiYH4qAAoJEFrkIMwCCZieVAAP
+/110c/v8MJzjo2lCNXT5KPawU9G6m6QRj1yN/ahm3UiR44fHHDEe5heddaJVKUll
+/MdJeWKRznn2VXLHE/0WGNzWXJlPubbcV9dEdAoGGnuiQ0k68ar2AuAsriBe9Wu/
+dJ7IXvrGsu/A9EnU1mNNca93a60x7O+9R9xUjszfDjwreA+fMCxht4tsXcQ1SR/+
+huUXGoR8Gg6FTMhAmKbs80zJeFRVZtj5/k/rlttJPei4/+o2+Tlkelv/2c0AmK57
+rgbyHp580MCSD+NOYmG2K28PkVIknKHFXWCmt7rXnH7T06baWiwGE9FcUo4aZcTE
+RaM5m619kCUioVObxbeR7Tn1pe2X6D2I7nU66qCugEUCwkXD3QEfrG8Cy5LDRnKe
+80cOzvwV4gD4SFY3BOOucfIWoWdggY7nRcCd9o6I32IaBKKmsJfow59jVy7Kmu6R
+Dxsg6FTvWoqupM5ymlVAkfqD5rHQDxAkzeRWNEdMd4WJnzNDRxa611LsepPfennW
+VIyKYKXim4vmZHsJB9oVgvxyEIaeYpXluHyQe4vb6Znj5YoljsVmMSTQEnDEAlKL
+zpU9WEqHS9hOai9Nw4un6LrXg6dK7qDUfxdA7spVYYP+YBEMT21Fxt8fN02672sZ
+G/hu2m6gqq21NCY1wdpjD0DpYsxc32Yk8SThQDshPnFTwsGUBBMBCAA+AhsDBQsJ
+CAcCBhUKCQgLAgQWAgMBAh4BAheAFiEE1YI8rLR3GRysAHVVWuQgzAIJmJ4FAmOn
+lGQFCRqQjYAACgkQWuQgzAIJmJ6sJRAAu7m9O24KDRdFn0O8sya8JOJvxO6Ekq//
+ln/uoFFrJXFNojQW9GfD5R2GhKa6w33M4LjXinCAzKMMpnr2hBoGoQsNqPdcd1wU
+hLMgxJLIu6L0Za1WLl5STIuRipalpZaizVYtjjPJ0CQTvmEYBYrm0xamYGsbYUbQ
+/4rb3CN/cmJfW55zpe2rPW+qL8lKrmKaQ103H95Mp8GdgX7GXJsiorJ1KZATKWX0
+mpRzbi98Q9r+L3XRuLeOJACnbx7AN7MO8W21JR7kHzwLdORDDz8kXJ8RIZ3FfKTR
+6vY0ddblquWyQuKLLBopH+w2hybe6irZax6MNcl7QITpu+iU1icuqQFOSmTZnm4Y
+sC4kU23dT0HWuwJtPFEwIo1F5OIPhO515rdOyarIxZWvA7/DjTaCGJ/KR3eKDOpJ
+dLlhHey/WN736btmVmHEeWQ++Hjq12VFrax84tN1v64enZsfPtT9eUgN5yt3Ufym
++ni1CfcHsdcV4R1Ay2l2kNenreiLDDrgGHP7HOAHNHfx023KMsPsdbe1gGRSn1zP
+NPOaJASM/H/f40+wq/VLhW00i4qDTuylloanm+fuLO11YxolWPcJYMHTgEDA6yuM
+i38oEIBs74brz43Ey8+axAnoFDWL4jTkd49rZjwXW8n/h5i4iZqUPDOSrEXCLlas
+eW2wNh3HICnCwZQEEwEIAD4CGwMFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AWIQTV
+gjystHcZHKwAdVVa5CDMAgmYngUCXgAJDgUJFOkCKAAKCRBa5CDMAgmYnr+JD/wO
+tjs3OdeL+n2VoiZ984klQ8LL9PS64GCA/g5vywYZ9FQdsr6GaxU//SAZ7z3nB9w+
+hkI0mgOOikRi/FmC0nKH0+XAG8BPILFU7wNZ1Pfywt/dUxDeiAkdcKphZM9IEX9h
+PRogMCF4de7oGZeVihWY5m5bIwwvrCPBxOI+lqB79COdnS6cWoi6iu8lOBy5EvRN
+xGTQnItc1hrPcMq/77sR8Te9dPhInAt7fI9mrUtNkmg7EsdgwVjxFPILzfbsXxS0
+xYFomKBx5jTyaH456/ymUNpBR4IE2eM8R7cPkKQdPT5iVB7iKRiu8cm/VYhNbGEQ
+Jg5ekaGXE4m5UufcdpcxOEQd/1Mu4jNINXFkS90+iQlNbvMFbqIhVRNzoBkuDvTx
+5mypqq6e7irJGqklnp3WJC9XJ3r67eXJCoEu2qxJa08wo4X2oD97aYEsELz06tg4
+FD/IxAyM3S5pdbwTBcLHe4xrinyoQi8m7iOH7IZZx/i1Cvc4K2lXVqV9EbOuFC2U
+vXjmoV+xuoUg9F6+4MYuRFBhSUvWQGW1XTaJeh+zNzoRr70KZr8hYNwalESNDQvb
+//+aicC+Aw7IFE86hRotnxQD/9HoQ9z2tA0awFjTQS5KlQzyf8P82vY0P39xjDoz
+mHtO1veqb+W/KyH0YQBkMiqlPUeMYkRixD8XlYEvTc7BTQROuqFgARAAprAiO4dd
+X7PCYekdLhCmpTl6FQRsM9pfObBOjeenMeZ9R0jAIiaaoNkAIHQjZFlcUDezzjzo
+egGqhy/Hf8Zk4rwJ9ypfg4tjKdWtnWPG4NjXMjXVvyiaGFl9JwOLV3tXhcLRHRkl
+8RKE1Sl5ez3SJ7cw3NbEgeJ2esRHjcFULQbQpg4IND6ati/vUOKmP9v/ocpwuUL/
+2yBZzwPETc6/jeK5MZPtxWBD6zoyoCFmo/0SEORFIyUoNQsev5+AtNowQ4ERXhYP
+BbJkkbrk2usQZDlSaqAcDg3xu4Ln5ft26CT8BJhrR6l4WiqPrR9Q92e+Mo/mC+UK
+L4cmUZeOGX1f9z1niNNjKAhwgjPFAwiahLf5WVmRV0mbTeD2mdnJXBam9UYo8Waa
+gmJZYQ4Gk+Vk8akWnO2EUP3baUFsN80wAUlsKIWXA7By19A80CiSMrhCkIlgCJY1
+U9+WWJShGybKv4YuYuizNi8zHOiWIKzulXRtkpHlXcKaoYdQz3hv5iPOr+SNSDo3
+IKLuuNhEAtOreTkuvjASz3X0etGjAqbv8jOCBSH3PlNB4BEwdGY5KAEZdARXPm2c
+v50PW1oamO7B1+ZXaPGM6iH7bZqFaGyBIrBSetnzQyaEuTx4ES858atajsEe+74h
+1Mkhy3zjqdDFkznKNmiMtK3ea2sZCcnqM6EAEQEAAcLBfAQYAQgAJgIbDBYhBNWC
+PKy0dxkcrAB1VVrkIMwCCZieBQJntR34BQkiYH4YAAoJEFrkIMwCCZieaDcP/0Df
+2aOdxqfZ/QG9GkYg/w/GdWf89mE26w34w2QvBoSqfBzASA7SKC15WtLei2d73dBz
+Hpm5jmOzUghSRAY4HxEw5kaZWzrMRTtgzxpX9nVCZZmVfC5p/drgogw2LtwvPeZX
+v7oZBq+s9rONpglKvRm68wzJUHR5UvzuPds8GLpJ7Br9dOe+LC9BfFoThAui0hqB
+bxWBIcLlGvu3ado49lS+G7Ydu5/Z6bubZ3jE+kZWNiHmFDudnEj/SiWWjj7ihOxm
++467nlVU24lKYN+Hu8v+KgWC0zjvi1mV0ZQTKD13Nl4+H2pcPkWw7KhVCQR7qyvq
+QPxUGRk2U0tOnu0nAlOmUsEc9K4Xa3m2Dzk6VBjf6MQHt2p2ZvGeDWda7FiX/fFH
+U8k6JsMveIzzipIc2apzvYvH1luHyiNg503H/2siaUCuCMp/w/xKYRjwrqW83Cla
+PpBn3IxNobEa0a0B0kWTR889AHkGBzePXB0zjgrl5+L6rV6BWkgN8NNdnJpMHLDT
+LAUTEEwX7EUw9JFvZ2tAL7K8s2fgryIINFftW/US6J7+I2n4n0udxaqma5xiZer+
+h/pJGLeFs8jUpOqMVhaW1s408mNsvV4JlB+1FFN9V1D7SUF/nSvwr8wFI17+WQiI
+Eo4jkeWmdGIJPXimapQ/O7XHB2IjB1y/xpynQzGUwsF8BBgBCAAmAhsMFiEE1YI8
+rLR3GRysAHVVWuQgzAIJmJ4FAmOnlIMFCRqQjaMACgkQWuQgzAIJmJ6hUBAApnDK
+sdUQg+zBmK1tJ7TiunZkvltKz/9di4guXQSm5gsMF5vKkyeED5ZBN+q33TzHXMKG
+noxmrMvq98UVDlWDvfOHLKGF+RczjeMYvbDk235bjFRvEG+2d6xu9Wp4PL599evd
+9hSlRkkrAkrCoHkI8NrWgEa51KEOOmRDgvC92yrCCxrhODfNpb1Pc9Ksd83Uyl5A
+6i6zRZK0yeFBM4UfZG7OymoDFQFWoGnxw/Pf5TS9fWaeOIggIfTKERrMHPuUNMHW
+NBsZBB5/eNBs3g9dqfCeBXecCtj+b1pxiW2Yv82B3G8rJEkc05V7clYNDwDbc2X1
+OUyZMpqmMkt7wfnx0OzwebdicRaiHU268mu3fKCnWGIy0biZynDUoY3ToFqGDuCn
+vMghjAF1Ych9D5IWevU7S0CwB60gUSiGsRBruOz8B3QizG0jb09BmztwIe4R1i/9
+Xcwpox4G94PmMoYDY44TVR/imewxKYrw2y3BmqI13yI48H21PK29IVRetQ7NbN8X
+/tTpHpIJKv94KnHuf2EeAniMdqnrfOK6UOCyq/AF4DyRBUMuCLo0ZhjRO7Yi/We5
+MxIH3oP7h569VMfjFpPTYA+2KDCxiMQerrGn4FsVj92h7D97E6hIz4GVKMZoQHaJ
+yJ9yjGvhcRecmPnxYL5hef5R5xUKcc5EMyr3vuDCwXwEGAEIACYCGwwWIQTVgjys
+tHcZHKwAdVVa5CDMAgmYngUCXgAJ8AUJFOkDEAAKCRBa5CDMAgmYniXhD/9i2z9h
+PaQFRi3WuU/fLfaiZRzL6uc7Fwf94G9DOr95lT3eaPdmVR8D8vOk4cmsGVfUcJX1
+lxrJWLt467hApigRzhmEiSjHrcszZmNQ+5pvGED8RHscGyb4lJwWjU5179OYYKpV
+cym8WEVHauDJTk9QSTBKF/nJVxJ/bQSrylw4c01z+TA0S4jTWQ/ScxygsJRa7iha
+LP8KTmpTs+6KUlG5G+tb4v+mQ4+fiYoFIUHk3vjxFAD+dzPHabZdIM2RGeTzMuK9
+7F/F8Qo0Vx9KTA1WAYGTYeh06tb5MmIiYFQR/KKJ7J6JcFvyQB6kBeVGELTNaROD
+gVGZbvicqVjX3I4fVboC3vWw0xtEOf0CWl9YwgJyEMC/sZDhv+hzEwTK/AW16lr+
+tw7jGTqFUrmbzQ4JpcWiLqIJkG3rLl/bjyTUVLcZuDBuu+zn0uUjl4n0TY2qvKj1
+anHTeLx4QVXH9nRQ7Uh47AtoAKddv7aeRWuI6vYjmqfNL2tyzVL8jVx7EsJ8cuI9
+TPqFGFoB7R+oY+KbzeVLWif+auWeLeVNsRNMaDPlCqVmfrKaeIThE3WXhwEoufOi
+2J4CY4DiNe/PX/P8gk+Dxtz+smr+kSrGJmEzRu13xOVqZ9tyVewdisT90hXkE5Df
+7BNYX7UHchtjQwyc14dsCeVf70WS3VQQlBaDPcLBZQQYAQIADwIbDAUCWHK01wUJ
+D1ut9wAKCRBa5CDMAgmYnsJaD/0dOb3gYBHXP+Sdh/QeH3UhaIQgKvBqFkQs5xXV
+11aQ3GYlqe2FSMXfqSYx2LLUmonxsLSw8bQbHbaQgs2/innLpghTea9Pu1aJYmDa
+85RFi4vruhht8k7k9fdJLY+L3r6+uL0o463IMZZhBFitBpvCabJUw0DUv0ZUp6Ff
+uRA99VwYXy+MbtY6+EqdcF3t/pmiYOcXWPEthJAeJ2zwxF+8dA/uvXjfVyb6NAbM
+uYyHYzaZmCYW+Bz3+LS8ct3QDOqi2L0cj3vbpLcOQD2WzS1tMqx4o8GzVCn0W024
+6y3rMURYBrd/4wGpSLogbyPk/HbUH+UqVmrOHeVXUp88oP91o/nU+8zP5y02Yc3J
+qllqT8275m1w3NuSwaTa6PiHCmT542OfpBNpPuhQ7IPPmwK5unOgFtQH8EydnMop
+uQjQeNHgrFKmfifn2++aTYnUdOV16qVy4oEvaBjF+RqFDkTtSbB4ji72JCpEAtiX
+StouSWU9K8eIi+Su5J31/jBJV8xyf+MZ3aAEBy3K3LSty7l4Ie7XqsMD8HMxXUQU
+j7rUEoysw1YZALeEIR7mzEW5+go+Lq/F71RPGcrDrdtKA5xNJZpoOB2IgS2ZBTlr
+jOdu9QGX8sZdr9I5SNsATtmOvlYHCsKiasQjyyIpbKrNT4ugmrTrkOXU85APrub8
+21Rp+sLBZQQYAQIADwIbDAUCV8FbjQUJDMkhLQAKCRBa5CDMAgmYnqzuEADIZcjC
+a8KasKDQd8dtaGLLUqJb2b2IpnWf4mh3OXlen1iYDGxsn2oBfDozrIKJT33Agk/r
+N7VucVDuGLqy/zjYYfpUX3cGiazTR8xFd70rtSJoQzAwsKGyCTIWiOgmilR3IoSV
+jfT+qiikyJ9StJA8eItiVviE5Infts2z62x1QVwpxjSIFH9Cz8cLOHuTfunu54E6
+y/iy3EdlFKNqSbqpFVt1c7pl2xH1N0fJ45Dghlh5FJ9ZHSVbe/UnToym623gntNs
+5BLDAoVmHnbnCYFRg7MgKsK7fQnNDX8wE/23ydTominpEtW2sJuEuRA99tf/eYqU
+3tfDzBl+s6mVwW+c9EYdqncSdP+TqtUlE1lvrqOoNX1UzNB19cnhiwRlZ7znO6kT
+8q0bH10Bb2bafdPaKBykqPP7uB3Jl90mI6/X95ZaH+EOEg3ZPuUxnKsf3m+hjA5o
+K4qDJwGRwyc2T644HKsi24al7/wuQ8Vaf/3cN5glyRSnIIWLkeJYGDN2zBKkS1dL
+TIAME1prPWnvGdBKq98H7XXPEj+v/zC3kn5x145gxWQne1n0ao8GzKE4WKkJzeHb
+u05ZDiqKjdTNQ+1jExXR3lIHfK3p5v0sLNdqyqXINmMImIo5Jz7QyIuSWS0qV1zw
+qco0Sj72TP51h/bIHQ9fu4OU4paU0msASNvHS8LBZQQYAQIADwIbDAUCUbSp3gUJ
+CJ2i/gAKCRBa5CDMAgmYnhQzEACMBnOJm0pfi/CUAL9pTl8jWv3zWIe0bEdR4iE5
+wglZ50tMiq0hAXCfWNDc4DnCKExWk7Z3oFIA9+HLEuxdVUsmoSXk/Nomd959XXfz
++uD6JnwrrDkZq3UHXUiwW1Ycio7Jgd5yADcjEmii3jHckrRBdzl8AG2SjTADu2Mx
+VT0sgWLY+NHHiZcBDW0F9FRwwrIVMqU07OQ+G4/uYfbrq8kP0ECw2AvlV84S8hW0
+Z2icEOy4y8ToFkxkY8nxNVG2BJI0DNBt4g/7KQauJ1lhtEl4eI8OJ/s67W4yv5Q5
+i39ngcUkvOT25RLGAh8OGc6lqwSy2vFkwHj92zzqqoDAKhyYuZNH6dHTuJfDTsOx
+OR9ZEGpErfH/RaUAELtif5vkBzmbqURZE8aNmwy+l9rliESo5yFtLgdwS8byGHCW
+Pj1Ix0cbB3m90ixtzpJ2njqbbZV3hKKREMIWdxG8/4/ZEo0h8oRd9BBeCLBjKL3X
+/mCPrcWqjlXbsMXUO3hzTrwMWyTziyMUh2f9D/CcLm0fvGubMteR8+A3+mjdGkJ7
+5+akDXv+PoJqxUzFwJNI2lONg+Aa7NQSMhqn+LnwR83wB97/2+jAeYeHIWaNpvZn
+Hnai1U1/wIqZtWtuNtcYcPrOaNSow1Jt4kvID0jvDtWKj6Xd9Fwv2NYYs5bJ4kAe
+VBqgYMLBZQQYAQIADwUCTrqhYAIbDAUJA8JnAAAKCRBa5CDMAgmYnkTID/4v9a/9
+gzUQYWViNxMigvznFLrB3mKSELlEwdyREmKAATaM0uE28xW8fmRPC2L7GB60dW9d
+5f3YmfdOZeK+gHqT78bi6gNUC0//F/fyjLKvc6q/TLDtwtYQDJ3t8feLRUUTkxWX
+ralOje8rkofIa+WBRXzeqzn0TX1AbetOeHiu3XA8GmyS8/c0qG9z0iF+BD4tFZDm
++nkP0diPpjUO34KZrTSWIjeRQDLetTDq/nEtlWHmPOt4wfu7GKWHNKTvfJb/LMBt
+f8eHxPFgZVPxWILpg4BESMmqdLB5sklcPvYysvsuFrh62edUxDj6yZWwfOAlih67
+Dvp16QBIdIcA6zrjv+rBD1fYxP401v8cCHQ433B0Y8Rr9mHzEq8DB2at+DUs45Ak
+2qxiZHnbk2RcVWevCxv3iOvbOTRk52YRlfbiN8UDNroSThMLcbXijXw+HGf9l7mI
+wbunoRC+NI40PhFF7wl76i6zDyot9/FJRJNT//Oqu+N8Uupf1AuAfnUj5IlJ7vLv
+GSpG6ZFNMmDcreHOCoJpw4Ir7o8iAzii+PEIylGubR83+sLkPQh8wZ5dmKTQ64MH
+wSHGCtWMMTPm5mnOWQb3ffQmWaOT3vwAhteVIb5hUHFlVNCj8YWWtQpq+vjn5LRC
+24/p9nTF/b/aKIDi9JbP67vhzq5eZLA95Ww6bw==
+=UIxp
 -----END PGP PUBLIC KEY BLOCK-----
diff --git a/znc.spec b/znc.spec
index 1c03452..8888f77 100644
--- a/znc.spec
+++ b/znc.spec
@@ -22,6 +22,10 @@ URL:            https://znc.in
 Source0:        %{url}/releases/archive/%{name}-%{version}.tar.gz
 Source1:        %{url}/releases/archive/%{name}-%{version}.tar.gz.sig
 
+# Refreshed from keyservers/WKD (same keys, newer signatures):
+#   sq --home=none network search --output - D5823CACB477191CAC0075555AE420CC0209989E | sq keyring filter --experimental --cert D5823CACB477191CAC0075555AE420CC0209989E --output update.pgp
+#   sq keyring merge --output gpgkey-5AE420CC0209989E.asc.new gpgkey-5AE420CC0209989E.asc update.pgp
+#   mv gpgkey-5AE420CC0209989E.asc.new gpgkey-5AE420CC0209989E.asc
 Source2:        gpgkey-5AE420CC0209989E.asc
 # Use system-wide crypto policy
 # https://fedoraproject.org/wiki/Packaging:CryptoPolicies
-- 
2.56.0


